pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -115,13 +136,6 @@ func (nc *nodeConfig) validateNode() error {
 	return nil
 }
 
-// getInstanceID gets the instanceID of VM for a given cloud provider ID
-// Ex: aws:///us-east-1e/i-078285fdadccb2eaa. We always want the last entry which is the instanceID
-func getInstanceID(providerID string) string {
-	providerTokens := strings.Split(providerID, "/")
-	return providerTokens[len(providerTokens)-1]
-}
-
```

Comment:
Please don't move code around across commits. Put in the location where you want and leave it there. You are causing unnecessary code churn and forcing the reviewer to look at the same code multiple times.